safekeeper/src/timeline.rs

```diff
@@ -101,7 +103,14 @@ impl SharedState {
     ) -> Result<Self> {
         let state = SafeKeeperState::new(zttid, peer_ids);
         let control_store = control_file::FileStorage::create_new(zttid, conf, state)?;
-        let wal_store = wal_storage::PhysicalStorage::new(zttid, conf);
+
+        let segment_complete = watch::channel((Lsn::MAX, Lsn::MAX));
+        let lsn_persisted = watch::channel(Lsn::MAX);
```

Comment:
Feels like `channel(Lsn(0))` is a more sane default choice for what was persisted?